### ISSUE TITLE
DT 5730 - Ultrahigh rate

### ISSUE THREAD

**Issue by xmatter** [zach-hartwig/ADAQAcquisition#3]
After I select the ultrahigh rate and start acquiring data, it still updates the digitized waveform with the 5730.

**Comment by zach-hartwig** [comment]
Thanks for reporting this. 

I was not able to test your issue on the hardware today but, from looking at the code just now, it appears that the issue you are having is indeed a bug in the code. I will try to confirm this issue on the hardware tomorrow. If confirmed, I will make the simple correction and push it to the master branch. This fix will be included in the very-near-term release of version 1.0.0.

**Comment by zach-hartwig** [comment]
I was able to confirm this issue on the hardware this morning. A fix has been implemented and pushed to the master branch. Waveforms will only be plotted in "Continuous" acquisition display mode and *not* in "Updateable" or "Nonupdateable" mode to maximize CPU efficiency. Plotting individual waveforms in "Updateable" mode would be nice but is difficult given the current coding structure. Thanks for reporting the issue, closing this at this time.
